specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -370,7 +391,7 @@ def coset_for_cell(cell_id: CellID) -> Cell:
     roots_of_unity_brp = bit_reversal_permutation(
         compute_roots_of_unity(FIELD_ELEMENTS_PER_EXT_BLOB)
     )
-    return Cell(roots_of_unity_brp[FIELD_ELEMENTS_PER_CELL * cell_id:FIELD_ELEMENTS_PER_CELL * (cell_id + 1)])
+    return CosetEvals(roots_of_unity_brp[FIELD_ELEMENTS_PER_CELL * cell_id:FIELD_ELEMENTS_PER_CELL * (cell_id + 1)])
```

Comment:
Something wrong here with the function wanting to return `Cell` but we are returning `CosetEvals`

Reply:
Pushed  bc607ec in an attempt to clarify things